src/overview/reducer.js

```diff
@@ -5,17 +5,23 @@ import * as actions from './actions'
 const defaultState = {
     searchResult: {rows: []},
     query: '',
+    isFetching:0,
 }
 
 function setQuery(state, {query}) {
     return {...state, query}
 }
 
 function setSearchResult(state, {searchResult}) {
-    return {...state, searchResult}
+    return {...state, searchResult, isFetching:1}
+}
+
+function playLoading(state){
+	return {...state, isFetching:0}
```

Comment:
Yeah initally I got the idea of setting the isFetching to true. But according to the conversation we had on the issue. It was given that multiple searches might take place simultaneously and that is why a counter would be more appropriate. I will change it immediately if required.